Mafia/src/main/java/com/example/Mafia/service/UserService.java

```diff
@@ -0,0 +1,50 @@
+package com.example.Mafia.service;
+
+
+
+import com.example.Mafia.model.User;
+import com.example.Mafia.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class UserService {
+
+private final UserRepository userRepository;
+    @Autowired
+    public UserService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    public Optional<User> findById(Long userId){
+        return userRepository.findById(userId);
+    }
+
+    public List<User> findAll(){
+        return userRepository.findAll();
+    }
+
+    public User createUser(User user){
+        return userRepository.save(user);
+    }
+
+    public User updateById(Long userId, User user) {
+        Optional<User> updatableUser = userRepository.findById(userId);
+        User newUser = null;
+        if (updatableUser.isPresent()) {
```

Comment:
Does it work as a PUT or as a PATCH though?

Reply:
More like PUT, but I don`t sure that yours variant work like PATCH